reference/test/distributed/index_map_kernels.cpp

```diff
@@ -195,3 +196,108 @@ TEST_F(IndexMap, CanGetLocalWithCombinedISWithInvalid)
     gko::array<local_index_type> expected(ref, {2, 3, 0, 1, 2, 4, -1, 1});
     GKO_ASSERT_ARRAY_EQ(local_ids, expected);
 }
+
+
+TEST_F(IndexMap, CanGetGlobalWithLocalISWithInvalid)
```

Comment:
```suggestion
TEST_F(IndexMap, CanGetGlobalWithLocalIsWithInvalid)
```
Is it supposted to be `IS`?

Reply:
I think I'm usually using `IS` in these tests to denote the `index_space` enum. I would keep it like that.